[BUG] <Shiro> ShiroProperties.getWhiteList() is null

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

ShiroProperties.getWhiteList() is null.

### Expected Behavior

_No response_

### Steps To Reproduce

_No response_

### Environment

```markdown
ShenYu version(s): latest
```


### Debug logs

Caused by: java.lang.NullPointerException: null
	at org.apache.shenyu.admin.shiro.config.ShiroConfiguration.shiroFilterFactoryBean(ShiroConfiguration.java:75) ~[classes/:na]
	at org.apache.shenyu.admin.shiro.config.ShiroConfiguration$$EnhancerBySpringCGLIB$$f1ce345f.CGLIB$shiroFilterFactoryBean$1(<generated>) ~[classes/:na]
	at org.apache.shenyu.admin.shiro.config.ShiroConfiguration$$EnhancerBySpringCGLIB$$f1ce345f$$FastClassBySpringCGLIB$$7a4cd641.invoke(<generated>) ~[classes/:na]
	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:244) ~[spring-core-5.2.2.RELEASE.jar:5.2.2.RELEASE]
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:363) ~[spring-context-5.2.2.RELEASE.jar:5.2.2.RELEASE]
	at org.apache.shenyu.admin.shiro.config.ShiroConfiguration$$EnhancerBySpringCGLIB$$f1ce345f.shiroFilterFactoryBean(<generated>) ~[classes/:na]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.base/java.lang.reflect.Method.invoke(Method.java:566) ~[na:na]
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:154) ~[spring-beans-5.2.2.RELEASE.jar:5.2.2.RELEASE]

### Anything else?

_No response_

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -62,7 +62,7 @@ jobs:
           java-version: ${{ matrix.java }}
       - name: Build with Maven
         if: env.SKIP_CI != 'true'
-        run: ./mvnw -B clean install
+        run: ./mvnw -B clean install -Prelease
       - uses: codecov/codecov-action@v1
         if: env.SKIP_CI != 'true'
 

--- a/shenyu-admin/pom.xml
+++ b/shenyu-admin/pom.xml
@@ -246,18 +246,26 @@
             <scope>runtime</scope>
         </dependency>
     </dependencies>
-
+    
+    <profiles>
+        <profile>
+            <id>release</id>
+            <build>
+                <resources>
+                    <resource>
+                        <directory>src/main/resources</directory>
+                        <excludes>
+                            <exclude>*.yml</exclude>
+                            <exclude>logback.xml</exclude>
+                        </excludes>
+                    </resource>
+                </resources>
+            </build>
+        </profile>
+    </profiles>
+    
     <build>
         <finalName>shenyu-admin</finalName>
-        <resources>
-            <resource>
-                <directory>src/main/resources</directory>
-                <excludes>
-                    <exclude>*.yml</exclude>
-                    <exclude>logback.xml</exclude>
-                </excludes>
-            </resource>
-        </resources>
         <plugins>
             <plugin>
                 <groupId>org.springframework.boot</groupId>

--- a/shenyu-bootstrap/pom.xml
+++ b/shenyu-bootstrap/pom.xml
@@ -410,18 +410,26 @@
         </dependency>
         <!--shenyu param mapping end-->
     </dependencies>
-
+    
+    <profiles>
+        <profile>
+            <id>release</id>
+            <build>
+                <resources>
+                    <resource>
+                        <directory>src/main/resources</directory>
+                        <excludes>
+                            <exclude>*.yml</exclude>
+                            <exclude>logback.xml</exclude>
+                        </excludes>
+                    </resource>
+                </resources>
+            </build>
+        </profile>
+    </profiles>
+    
     <build>
         <finalName>shenyu-bootstrap</finalName>
-        <resources>
-            <resource>
-                <directory>src/main/resources</directory>
-                <excludes>
-                    <exclude>*.yml</exclude>
-                    <exclude>logback.xml</exclude>
-                </excludes>
-            </resource>
-        </resources>
         <plugins>
             <plugin>
                 <groupId>org.springframework.boot</groupId>

--- a/shenyu-dist/shenyu-admin-dist/pom.xml
+++ b/shenyu-dist/shenyu-admin-dist/pom.xml
@@ -62,7 +62,7 @@
         <profile>
             <id>release</id>
             <activation>
-                <activeByDefault>true</activeByDefault>
+                <activeByDefault>false</activeByDefault>
             </activation>
             <build>
                 <finalName>apache-shenyu-incubating-${project.version}</finalName>

--- a/shenyu-dist/shenyu-bootstrap-dist/pom.xml
+++ b/shenyu-dist/shenyu-bootstrap-dist/pom.xml
@@ -55,7 +55,7 @@
         <profile>
             <id>release</id>
             <activation>
-                <activeByDefault>true</activeByDefault>
+                <activeByDefault>false</activeByDefault>
             </activation>
             <build>
                 <finalName>apache-shenyu-incubating-${project.version}</finalName>

--- a/shenyu-dist/shenyu-src-dist/pom.xml
+++ b/shenyu-dist/shenyu-src-dist/pom.xml
@@ -31,7 +31,7 @@
         <profile>
             <id>release</id>
             <activation>
-                <activeByDefault>true</activeByDefault>
+                <activeByDefault>false</activeByDefault>
             </activation>
             <build>
                 <finalName>apache-shenyu-incubating-${project.version}</finalName>

